### PR TITLE
Documentation for GitHub Action for Soda

### PR DESCRIPTION
**Pull request by janet-can**
Documentation for integrating Action for Soda into a GitHub workflow.

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -212,6 +212,8 @@
     page: soda/integrate-alation.md
```
Comment (dirkgroenen) [inline _data/nav.yml:212]: I think we can also our quick-start-dev to contain the new Github Action, as opposed to the custom one. 

```yml
# This GitHub Action runs a Soda scan on a Snowflake data source called reporting_api_marts.
name: Run Soda Scan on [reporting_api__marts]
# GitHub triggers this job when a user creates or updates a pull request.
on:
  pull_request:
jobs:
  run:
    # The job runs on the latest Docker image for Ubuntu.
    runs-on: ubuntu-latest
    steps:
      # Step 1: Checkout the repository code into the container.
      - name: Checkout
        uses: actions/checkout@v2
      # Step 2: Run the Soda Library Github Action
      - name: Perform Soda Scan
        uses: sodadata/soda-github-action@v1
        env:
          SODA_CLOUD_API_KEY: $\{\{ secrets.SODA_CLOUD_API_KEY \}\}
          SODA_CLOUD_API_SECRET: ${{ secrets.SODA_CLOUD_API_SECRET }}
          SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
        with:
          soda_library_version: v1.0.4
          data_source: reporting_api__marts
          configuration: ./soda_checks/configs/configuration.yml
          checks: ./soda_checks/checks
```

File: soda/integrate-github.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          SODA_CLOUD_API_KEY: ${{ secrets.SODA_CLOUD_API_KEY }}
+          SODA_CLOUD_API_SECRET: ${{ secrets.SODA_CLOUD_API_SECRET }}
+          SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
+          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
+        with:
+          soda_library_version: v1.0.4
+          data_source: snowflake
+          configuration: ./configuration.yaml
+          checks: ./checks.yaml
+```
```
Comment (dirkgroenen) [inline soda/integrate-github.md:37]: By adding `raw` we make sure Jekyll doesn't escape the curly braces in the `secrets` part. 

```
{% raw %}
```yaml
name: Scan for data quality

on: pull_request
jobs:
  soda_scan:
    runs-on: ubuntu-latest
    name: Run Soda Scan
    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Perform Soda Scan
        uses: sodadata/soda-github-action@v1
        env:
          SODA_CLOUD_API_KEY: ${{ secrets.SODA_CLOUD_API_KEY }}
          SODA_CLOUD_API_SECRET: ${{ secrets.SODA_CLOUD_API_SECRET }}
          SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
        with:
          soda_library_version: v1.0.4
          data_source: snowflake
          configuration: ./configuration.yaml
          checks: ./checks.yaml
```
{% endraw %}
```

File: _data/nav.yml (hunk 1 above)
Comment (janet-can) [reply _data/nav.yml:212]: Absolutely! It's on my list. And also to add in the bit about accessing the report and scan logs in cloud.

File: soda/integrate-github.md (hunk 2 above)
Comment (janet-can) [reply soda/integrate-github.md:37]: Ugh. Jekyll md strikes again. I'll be sure to comb through any of those.

**Pull request merged**